portfolio/src/main/webapp/index.html

```diff
@@ -92,7 +92,7 @@ <h1>Michaela Murray's Portfolio</h1>
     <label id="location-label">Places I want to go:</label>
     <select name="locations" id="locations" onchange="loadMap()">
         <option value="CERN">CERN, Geneva, Switzerland</option>
-        <option value="Ramen">Ramen Nagi, Palo Alto, California, US</option>
+        <option value="Ramen-Nagi">Ramen Nagi, Palo Alto, California, US</option>
```

Comment:
You'll also need to update your check in `loadMap()` to deal with the new value. Note that this is a risk when keeping data in two spots. I'm fine with this for now, but might be a good idea to have all this data coming from the server response... also just something to keep in mind moving forward.

Reply:
ah ok just updated! I'll add that as a todo on my list :)

Reply:
Alternatively you could add a new js class that holds the list of locations and their coordinates. Then use that class to generate your select options an also use the same class to get the coordinates from the selected option.